src/utilities/findBreakingChanges.ts

```diff
@@ -103,10 +129,10 @@ export function findDangerousChanges(
   );
 }
 
-function findSchemaChanges(
+export function findSchemaChanges(
```

Comment:
Should we change the name of the file now or when we remove the old functions? Could be later…

Reply:
I was hesitant due to it also being a breaking change but generally supportive

Reply:
Gotcha, in that case I'll add the name change